sdk/servicebus/service-bus/src/core/linkEntity.ts

```diff
@@ -103,14 +145,15 @@ export class LinkEntity {
     this.address = options.address || "";
     this.audience = options.audience || "";
     this.name = getUniqueName(name);
+    this._logPrefix = `[${context.namespace.connection.id}|l:${this.name}|a:${this.address}]`;
```

Comment:
Is the use of `l:something|a:something` we are standardizing on? If yes, we should log an issue with `blocking-release` label to update the logs all across the package so that we dont have a mix of different formats

Reply:
(will file an issue for standardization of whatever we decide)

How do you feel about the format as it is? My real goal is just to have the IDs be easily scannable rather than being embedded in the log message itself.

CC: @HarshaNalluru

Reply:
The intention is good, the log message is also cleaner. We can go with the new format everywhere.

Reply:
Filed remaining log consistency work as #10638